website/content/en/docs/contribution-guidelines/releasing.md

```diff
@@ -28,6 +28,22 @@ Install GNU `sed` and `make` which may not be by default:
 brew install gnu-sed make
 ```
 
+Verify the version of `make` using command `make --version`.
+
+Here, `brew` automatically installs `3.81` version of `make`. In order to update it to `4.3` use below command.
+
+```sh
+PATH="/usr/local/opt/gnu-sed/libexec/gnubin:$PATH"
```

Comment:
this is for adding `gnubin` to PATH, right ?

Reply:
yes. this will upgrade make version.